src/dbus.hpp

```diff
@@ -12,6 +12,9 @@
 class Dbus
 {
   public:
+    // Default ethernet interface used to manipulate wth DNS/NTP servers
+    static constexpr const char* defaultEth = "eth0";
```

Comment:
I propose to define it as a meson option.
Different platforms may have different default interfaces. 
I'm not fully sure, but it seems to me, on Vegman it is eth1.
And it would be nice to have a possibility to define it at configure/compile stage.

Reply:
Done!